include/ddc/periodic_sampling.hpp

```diff
@@ -127,8 +131,11 @@ class PeriodicSampling : detail::PeriodicSamplingBase
         {
             return m_origin
                    + continuous_element_type(
-                             static_cast<int>((icoord.uid() + m_n_period / 2 - 1) % m_n_period)
-                             - static_cast<int>(m_n_period / 2 - 1))
+                             static_cast<int>(
+                                     (icoord.uid()
+                                      + (m_n_period - static_cast<int>(m_odd_period)) / 2)
```

Comment:
About the computation `(m_n_period - static_cast<int>(m_odd_period)) / 2` is `m_odd_period` really necessary ? I tend to think the integer division already handles it fine ?

Reply:
Ah yeah that's correct, thx